src/core/ComplexFilter/index.tsx

```diff
@@ -18,23 +18,29 @@ export type ComplexFilterValue =
   | DefaultMenuSelectOption
   | null;
 
-interface ComplexFilterProps {
+export type Value<T, Multiple> = Multiple extends undefined | false
+  ? T | null
+  : Array<T> | null;
+
+interface ComplexFilterProps<Multiple> {
```

Comment:
Okay, so I get that `Multiple` is a type param here and so it has ripple effects throughout the interface, but I don't understand how it actually gets used with the real props for the component definition...?